tests/unit/test_notifications.py

```
@@ -0,0 +1,292 @@
+    def __iter__(self):
+        yield from self.to_dict().items()
+
+
+async def _mock_get_db():
+    yield make_mock_session([])
+
+
+def _make_auth_override(tenant_id: int = 1, user_id: int = 99):
+    """Mimics real JWT-based AuthContext creation: raises 401 for invalid tenants."""
+    if not isinstance(tenant_id, int) or tenant_id <= 0:
+        raise HTTPException(status_code=401, detail="Token is missing a valid tenant_id")
```

Pylint (4.0.5)

[error] Undefined variable 'HTTPException'

(E0602)